rt_gene/src/rt_gene/extract_landmarks_method.py

```diff
@@ -34,29 +34,90 @@
 from rt_gene.msg import MSG_SubjectImagesList
 from rt_gene.cfg import ModelSizeConfig
 from rt_gene.subject_ros_bridge import SubjectListBridge
-from rt_gene.tracker import GenericTracker, TrackedElement
+from rt_gene.GenericTracker import TrackedElement
+from rt_gene.FaceEncodingTracker import FaceEncodingTracker
+from rt_gene.SequentialTracker import SequentialTracker
 
 import face_alignment
 from face_alignment.detection.sfd import FaceDetector
 
-import torch
+import dlib
```

Comment:
Can we make dlib a soft dependency, only required if `face_encoding_tracker == True`?

Reply:
Moved dlib dependency into FaceEncodingTracker